### ISSUE TITLE
Can't save empty array

### ISSUE THREAD

**Issue by MihaelIsaev** [vapor/postgresql#80]
I have a Fluent model

```swift
final class Experiment: Content {
    var id: UUID?
    var tags: [String]?
}
```

When I'm trying to save it
```swift
Experiment(tags: ["test"]).save(on: conn) // works
Experiment(tags: nil).save(on: conn) // works
Experiment(tags: []).save(on: conn) // throws error
```

error is

```
⚠️ [PostgreSQLError.server.error.array_recv: wrong element type]
```

It worked before update to the latest Fluent/PostgreSQL release with a lot of changes.
So it seems like a bug, cause saving empty array is not a crime 🤔

**Comment by tanner0101** [comment]
Fixed in #79, thanks for reporting!

**Comment by Jinxiansen** [comment]
Hi, brother, I have used MySQLModel, also have this problem, do not know when to solve the release version?
